core/templates/dev/head/app.js

```diff
@@ -30,9 +30,10 @@ var oppia = angular.module(
     window.GLOBALS ? (window.GLOBALS.ADDITIONAL_ANGULAR_MODULES || [])
                    : []));
 
-// TODO(sll): Get this to read from a common JSON file; it's replicated in
-// feconf.
-oppia.constant('CATEGORY_LIST', GLOBALS.ALL_CATEGORIES);
+for (var constantName in CONSTANTS) {
+  oppia.constant(constantName, CONSTANTS[constantName]);
```

Comment:
(I mean, the point is to test this for loop, and I believe a test with the actual real, json-loaded constants would address that just as well as a test with faked constants, especially since there doesn't seem to be a natural seam in place to insert the latter.)